Dockerfile

```diff
@@ -1,72 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
-
-# Downloads, builds and installs Apache Qpid Proton and Qpid Dispatch on CentOS and starts the dispatch router
-# /main is the top level folder under which proton (/main/qpid-proton/) and dispatch (/main/qpid-dispatch) source code is downloaded from github
-# /usr/lib64 is the folder in which the proton artifacts are installed.
-# /usr/sbin is the folder in which dispatch executable installed
-
-
-# Copy this docker file to your local folder. Build the docker image like this:
-# > sudo docker build -t <username>/dispatch --file=Dockerfile .
-
-# "<username>/dispatch" is a convention, you can call it whatever you want
-
-# To launch a container running the new image:
-# > sudo docker run -i -t <username>/dispatch
-
-# To launch a container running the new image with an interactive shell prompt:
-# > sudo docker run -i -t <username>/dispatch /bin/bash
-
-################# Begin code #######
-
-# Get the latest Centos version from dockerhub
-FROM quay.io/centos/centos:stream8
-
-MAINTAINER "dev@qpid.apache.org"
```

Comment:
why delete this?

Reply:
This is no longer an apache related build. Therefore I was going to keep just the "existing"
Dockerfile used to build the upstream image which is currently used by Skupper.

Any special reason to keep this one?

Reply:
If the skupper image is going to use quay.io/centos/centos:stream8, then no reason to keep this. Other Dockerfiles in https://github.com/skupperproject/skupper-router/tree/main/dockerfiles can also be purged, I think. But that raises the question whether it should be done in this PR, or in subsequent PR later.

Reply:
I will remove it now, but the others under dockerfiles will be left for an upcoming PR.

Reply:
I agree, no reason to keep this